### PR TITLE
feat: add log() method to BuildClient (JS)

### PR DESCRIPTION
**Pull request by tobice**
This wraps the new `/actor-builds/<BUILD_ID>/log` API.

### CODE REVIEW

**Comment by tobice** [comment]
I guess this shouldn't be merged / released before the API is actually updated? https://github.com/apify/apify-core/pull/14281

**Comment by fnesveda** [comment]
> I guess this shouldn't be merged / released before the API is actually updated? [apify/apify-core#14281](https://github.com/apify/apify-core/pull/14281)

Yep, and until the API reference is updated too.

**Review by fnesveda — APPROVED** [review]
Cool!

**Comment by tobice** [comment]
Keeping this `on hold` before the API is merged in https://github.com/apify/apify-core/pull/14281.

**Review by drobnikj — APPROVED** [review]
👍

**Pull request merged**
